math.big: Implement str() for big numbers

lilenvy opened the pull request:
Previously this was printing HEX.
Now it prints base 10 and the TODO was removed.

I also ran vfmt, I can revert the vfmt changes if you prefer, please let me know.

serkonda7 commented:
I have a small suggestion for your next PR. 
Do one commit where you run vfmt and another one with your changes. Then it's easier to review.

lilenvy commented:
Thanks for the advice. I have split the commit on 2 so hopefully it's easier to review, or should they be separate PRs in the future?

BTW I was getting these failures on a few systems which I didn't understand but I added explicit references (`&`) and I hope they succeed, they are making good progress I believe:
```
/tmp/v/test_session_169299294361/big_test.945489811125451239.tmp.c:12996:33: error: cannot take the address of an rvalue of type 'big__Number' (aka 'struct bn')
        string _t345 = big__Number_str(&big__from_u64(4294967295));
```

serkonda7 commented:
Thanks, nice job. 
One PR but separate commits is fine.
